heartbeat/heartbeat.go

```
@@ -0,0 +1,52 @@
+		fmt.Printf("Sending heartbeat\n")
+		if err := sendHeartbeat(); err != nil {
+			fmt.Printf("Heartbeat error: %s\n", err.Error())
+			log.Printf("Heartbeat error:: %s", err.Error())
+		} else {
+			fmt.Printf("Heartbeat sent successfully.\n")
+			log.Printf("Heartbeat sent successfully..")
+		}
+		fmt.Printf("Finshed sending heartbeat\n")
+		<-ticker.C
+	}
+	fmt.Printf("Exiting runHeartbeat\n")
```

golangci-lint (1.62.2)

unreachable: unreachable code

(govet)